distribution/docs/src/main/resources/content/_configuring/reusing-configurations.adoc

```diff
@@ -7,7 +7,7 @@
 
 == {title}
 
-The Configuration Export/Import capability allows administrators to export the current ${branding} configuration and use it to restore the same state for either a brand new installation or a second node for a highly available deployment.
+The Configuration Export/Import capability allows administrators to export the current ${branding} configuration and use it to restore the same state for either a brand new installation or a second node for a Highly Available Cluster.
```

Comment:
❓ @ricklarsen, how should we handle the case where the console has been disabled during hardening? Should we mention this here or just in the hardening section?

Reply:
We would have to mention it all throughout the documentation whenever we talking about the command line if we're going to do that. I think mentioning it in the hardening guide is sufficient.